src/IECoreDelight/Renderer.cpp

```diff
@@ -281,7 +281,7 @@ class DelightOutput : public IECore::RefCounted
 			driverParams.add( { "drivername", &typePtr, NSITypeString, 0, 1, 0 } );
 			driverParams.add( { "imagefilename", &namePtr, NSITypeString, 0, 1, 0 } );
 
-			m_driverHandle = DelightHandle( context, "outputDriver:" + name, ownership, "outputdriver", driverParams );
+			m_driverHandle = DelightHandle( context, "outputDriver:" + output->getName(), ownership, "outputdriver", driverParams );
```

Comment:
Amazing how simple this is! Might be worth a comment stating that `NSICreate` will just return an existing node if the handle already exists and that's how we coalesce multi-part. I only learned about that detail very recently.

Reply:
Sounds good! Will update the code with the comment.